README.md

```diff
@@ -1,2 +1,26 @@
 # fobnail-test-environment
-Fobnail test environment
+
+## Usage example
+
+1. Download and prepare the testing repository by opening the terminal and
+    running the following commands:
+
+    ```bash
+    git clone git@github.com:fobnail/fobnail-test-environment.git
+    cd fobnail-test-environment
+    virtualenv -p $(which python3) robot-venv
+    source robot-venv/bin/activate
+    pip install -U -r requirements.txt
+    ```
+
+1. Set the absolute Fobnail path on your device in the `variables.robot` file:
+    
+    ```bash
+    ${absolute_fobnail_path}                        /home/user/fobnail
```

Comment:
I think, that instead of setting each time this path in the file, we should specify it as a variable in the robot command.

Reply:
I agree, remove variable from `variable.robot`